examples/example2/script.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Initialize Mollie Components instance
+ */
+const mollie = Mollie("pfl_saaa222aaa3", {
+  locale: "en_US"
+});
+
+/**
+ * Error containers
+ */
+const cardNumberErrors = document.getElementById("card-number-errors");
```

Comment:
My bad, but this should be singular: error

Reply:
Updated this